VAMobile/src/screens/HealthScreen/SecureMessaging/ViewMessage/IndividualMessageErrorComponent.tsx

```diff
@@ -40,7 +40,7 @@ function IndividualMessageErrorComponent() {
               </Box>
```

Comment:
Could potentially be partially simplified moving this to the `primaryButton` or `secondaryButton` props to AlertWithHaptics.